software/perception/matt_sandbox/matlab/multisense_calib/test_script_vicon.m

```diff
@@ -121,7 +121,7 @@
     end
     channel = event.channel;
     if (strcmp(channel,'BDI_UTIMES'))
```

Comment:
Agreed. I presume this is a currently unused script and I only fixed the atlas_behavior_command_t since it was the same go as fixing it in Director as required for the videoPlayer. We can revisit this script once we get the Vicon system and need it or once MIT uses it again